metadata.rb

```diff
@@ -8,8 +8,8 @@
 issues_url 'https://github.com/cvent/octopus-deploy-cookbook/issues'
 version '0.6.5'
 
-depends 'windows', '~> 1.38'
-depends 'windows_firewall', '~> 3.0.2'
+depends 'windows', '>= 1.38.0'
```

Comment:
I am fine with removing the constraint on this entirely TBH.  The windows cookbook is simply used to support legacy clients.